StreamTemp-Integ.py

```diff
@@ -68,13 +64,17 @@
 
         # Define root directory of database and output
         # Modify this based on your own location
-        rootDatabase = os.path.join(os.path.sep, absRoot, 'scratch', 'SNTemp')  # CAMELS dataset root directory: /scratch/Camels
-        rootOut = os.path.join(os.path.sep, absRoot, 'TempDemo', 'FirstRun')  # Model output root directory: /data/rnnStreamflow
-
-        forcing_path = os.path.join(os.path.sep, rootDatabase, 'Forcing', 'Forcing_new', f_list)  # obs_18basins
-        forcing_data =[]#pd.read_feather(forcing_path)
-        attr_path = os.path.join(os.path.sep, rootDatabase, 'Forcing', 'attr_new', a_list)
-        attr_data =[]#pd.read_feather(attr_path)
+        rootDatabase = os.path.join(os.path.sep, absRoot, 'in')  # CAMELS dataset root directory: /scratch/Camels
+        rootOut = os.path.join(os.path.sep, absRoot, 'output')  # Model output root directory: /data/rnnStreamflow
+
+        forcing_path = os.path.join(os.path.sep, rootDatabase, 'forcing', f_list)  # obs_18basins
+        attr_path = os.path.join(os.path.sep, rootDatabase, 'attr', a_list)
+        if os.path.exists(os.path.join(os.path.sep, rootDatabase, 'in', 'Statistics_basinnorm.json')):
```

Comment:
Joining rootDatabase (which is `[absRoot]/in`) with an additional `in` folder here...is that right? Seems sketchy

Reply:
good catch